src/managers/config.py

```diff
@@ -183,6 +186,18 @@ def kafka_opts(self) -> str:
             f"-Dcharmed.kafka.log.level={self.log_level}",
         ]
 
+        http_proxy = os.environ.get("JUJU_CHARM_HTTP_PROXY")
+        https_proxy = os.environ.get("JUJU_CHARM_HTTPS_PROXY")
+        no_proxy = os.environ.get("JUJU_CHARM_NO_PROXY")
```

Comment:
**question** there is no way to get this from `ops`? It seems weird that we need to access to some environment variable. I would have thought that ops would have provided some apis/bindings

Reply:
I asked this question at some point. I was told that's it at the moment, and that there is this issue opened:
https://github.com/canonical/operator/issues/1113

Reply:
All right! You are definitely ahead of me on this :D 

Thanks for the link! Hope they had at least a discussion in Madrid. The "messiness" mentioned in the issue worries me a bit that we may need to give some support/attention to the part above. But this is out of this PR scope. Happy to resolve this!